<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>双目摄像机三维空间定位系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>720</width>
      <height>370</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>15</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>双目相机</string>
     </property>
    </widget>
    <widget class="QLabel" name="stereoCamera">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>640</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>300</y>
       <width>335</width>
       <height>70</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>打开相机</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>相机关闭</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>打开图片</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>图片关闭</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>相机模式：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>图片模式：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_7">
        <property name="text">
         <string>拍照</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>360</width>
      <height>350</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>15</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>深度图</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>320</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>310</y>
       <width>90</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>保存深度图</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>310</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>进行立体匹配</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>380</y>
      <width>360</width>
      <height>350</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>15</y>
       <width>90</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>左图像测距</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>320</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>310</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>图像测距</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>310</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>保存测距图</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>450</width>
      <height>720</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>400</width>
       <height>660</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>15</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>数据显示</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>15</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
    <addaction name="actionb"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>点云图</string>
    </property>
    <addaction name="action_6"/>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_7"/>
    <addaction name="action_8"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionb">
   <property name="text">
    <string>保存深度图</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>保存点云图</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>打开双目相机</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>打开图片</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>拍照</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>系统介绍</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>相机参数</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>生成点云图</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
